clients/csharp-uwp/UWPVoiceAssistantSample/AppSettings.cs

```diff
@@ -62,12 +84,13 @@ public class AppSettings
         /// </summary>
         /// <param name="configFile">config.json.</param>
         /// <returns>Instance of AppSettings.</returns>
-        public static AppSettings Load(string configFile)
+        public static async Task<AppSettings> Load(StorageFile configFile)
         {
-            StreamReader file = new StreamReader(configFile);
-            string config = file.ReadToEnd();
-            file.Close();
-            AppSettings instance = JsonConvert.DeserializeObject<AppSettings>(config);
+            //StreamReader file = new StreamReader(configFile);
+            string file = await Windows.Storage.FileIO.ReadTextAsync(configFile);
+            //string config = file.ReadToEnd();
+            //file.Close();
```

Comment:
I'm not super familiar with c#/uwp file io- do we not need the closefile anymore?

Reply:
I used this page as guidance. Docs does not indicate closing of file and no close method exists on the file.

https://docs.microsoft.com/en-us/windows/uwp/files/quickstart-reading-and-writing-files

Reply:
Are the commented lines going to be removed?